pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -265,6 +278,28 @@ func (c *StaticResourceController) Sync(ctx context.Context, syncContext factory
 	errors := []error{}
 	var notFoundErrorsCount int
 	for _, conditionalManifest := range c.manifests {
+		ready, err := c.precondition(ctx, conditionalManifest.files)
+		if err != nil {
+			if _, _, updateErr := v1helpers.UpdateStatus(ctx, c.operatorClient, v1helpers.UpdateConditionFn(operatorv1.OperatorCondition{
```

Comment:
does this call have an internal comparison or will it continuously update?

Reply:
It does this continuously. I was copy-pasting this part from `apiservice_controller.go` BTW.